index.rst

```diff
@@ -654,8 +711,9 @@ change requests to avoid running variances wherever possible.
 Closing the Cycle
 -----------------
 
-Assuming everything has gone to plan, by the end of a cycle all epics should
-be marked as “done”. When an epic is marked as done, it is equivalent to
+Assuming everything has gone according to plan, by the end of a cycle deliverables
+for all epics should be verified, and all epics should be marked as “done”.
+When an epic is marked as done, it is equivalent to
```

Comment:
I'm adding these words, but I think it's vital that we define process for "verifying deliverables", otherwise it's pretty meaningless.